pybikes/data/encicla.json

```diff
@@ -0,0 +1,18 @@
+{
+    "instances": [
+        {
+            "tag": "encicla",
+            "meta": {
+                "latitude": 6.254942,
+                "longitude": -75.567982,
+                "city": "Medellin",
+                "name": "Encicla",
+                "country": "CO"
+            },
+            "feed_url": "http://encicla.gov.co/status", 
+            "format": "json"
```

Comment:
This format `json` was just a formality in bixi to support multiple feeds with an slightly compatible implementation. On this case, not knowing if there are more systems that share this syntax it's safe to just assume it's always going to be a json.